admin/timescale/create_views.sql

```diff
@@ -1,18 +1,4 @@
-BEGIN;
-
-CREATE VIEW listen_count
-       WITH (timescaledb.continuous, timescaledb.refresh_lag=43200, timescaledb.refresh_interval=3600)
-         AS SELECT time_bucket(bigint '86400', listened_at) AS listened_at_bucket, user_name, count(listen)
-            FROM listen group by time_bucket(bigint '86400', listened_at), user_name;
-
-CREATE VIEW listened_at_max
-       WITH (timescaledb.continuous, timescaledb.refresh_lag=43200, timescaledb.refresh_interval=3600)
-         AS SELECT time_bucket(bigint '86400', listened_at) AS listened_at_bucket, user_name, max(listened_at) AS max_value
-            FROM listen group by time_bucket(bigint '86400', listened_at), user_name;
-
-CREATE VIEW listened_at_min
-       WITH (timescaledb.continuous, timescaledb.refresh_lag=43200, timescaledb.refresh_interval=3600)
-         AS SELECT time_bucket(bigint '86400', listened_at) AS listened_at_bucket, user_name, min(listened_at) AS min_value
-            FROM listen group by time_bucket(bigint '86400', listened_at), user_name;
-
-COMMIT;
+CREATE MATERIALIZED VIEW listen_count_30day
+       WITH (timescaledb.continuous)
+         AS SELECT time_bucket(bigint '2592000', listened_at) AS listened_at_bucket, user_name, count(listen)
```

Comment:
I think timescale supports intervals like 30 days. Would be more readable to use that if possible.

Reply:
I sounds like it should be supported, but in reality if you use anything other than seconds, it fails. I really wanted to use month, rather than 30 days... :(

Reply:
Ah! ok. makes sense. Intervals like 30 days only work if the column is a date or timestamp.